src/components/paragraphs/sum-video/video-paragraph.tsx

```diff
@@ -0,0 +1,29 @@
+"use client"
+import { HtmlHTMLAttributes, useState } from "react";
+import Oembed from "@components/elements/ombed";
```

Comment:
Replaced Oembed with [react-youtube](https://www.npmjs.com/package/react-youtube)

Refer to the [iFramer API docs](https://developers.google.com/youtube/iframe_api_reference#Events) for additional information on how to implement the play for 5 seconds and pause